assets/javascripts/discourse/templates/components/assignee-chooser-row.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if this.item.isUser}}
+  {{avatar this.item imageSize="tiny"}}
+  <span class="identifier">{{format-username this.item.id}}</span>
+  <span class="name">{{this.item.name}}</span>
+  {{#if (and this.item.showUserStatus this.item.status)}}
+    <UserStatusMessage @status={{this.item.status}} @showDescription={{true}} />
+  {{/if}}
+  {{decorate-username-selector this.item.id}}
+{{else if this.item.isGroup}}
+  {{d-icon "users"}}
+  <span class="identifier">{{this.item.id}}</span>
+  <span class="name">{{this.item.full_name}}</span>
+{{else}}
+  {{d-icon "envelope"}}
+  <span class="identifier">{{this.item.id}}</span>
+{{/if}}
+{{#if this.item.isSuggestion}}
```

Comment:
I assume this is not yet using the new sk row index right?

Reply:
Unfortunately no because the text has to be displayed for the first element and the first search without a term.